.github/workflows/release-please.yml

```diff
@@ -0,0 +1,60 @@
+on:
+  push:
+    branches:
+      - main
+
+name: Run Release Please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v2
+        id: release
+        with:
+          command: manifest
+          token: ${{secrets.RELEASE_PR_TOKEN}}
+          default-branch: main
+
+      # The logic below handles the npm publication:
+      - name: Checkout Repository
+        if: ${{ steps.release.outputs.releases_created }}
```

Comment:
Is releases meant to be plural? I might be misremembering but i thought it was singular.

Reply:
Yeah I believe their README is incorrect. I submitted a PR to correct it: https://github.com/googleapis/release-please/pull/1057

Based on my output from this test run it's plural: https://github.com/willarmiros/test-package-lock-repo/runs/3617992025?check_suite_focus=true#step:3:4